tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -238,6 +263,13 @@ public:
 
    void Finalize();
 
+   // Helper functions for RMergeableValue
+   std::unique_ptr<RMergeableValueBase> GetMergeableValue()
+   {
+      std::unique_ptr<RMergeableValueBase> mergeableptr = std::make_unique<RMergeableFill<Hist_t>>(*fResultHist);
+      return mergeableptr;
```

Comment:
I don't think it can since I'm returning a pointer to the base class but creating instances of derived. e.g. with `RMergeableMean`:
`std::make_unique<RMergeableMean>(meanValue, countsValue)` creates a `std::unique_ptr<RMergeableMean>` but I actually need to pass an `std::unique_ptr<RMergeableValueBase>`. Unless it would automatically get converted in the return statement (I may be missing this information actually), I can't create it directly with `make_unique` because there is no right constructor for the base.

Reply:
I'm suggesting `return std::make_unique<Derived>(...);`. The fact that `std::unique_ptr<RMergeableValueBase> mergeableptr = std::make_unique<RMergeableFill<Hist_t>>(...)` compiles shows that unique_ptrs are implicitly covertible to unique_ptrs to base classes.

Reply:
Done in the latest commits